dispatcher/backend/src/routes/schedules/schedule.py

```diff
@@ -306,6 +302,23 @@ def patch(
                 # not allowed (yet) in UpdateSchema, only config keys can be set to null
                 setattr(schedule, key, value)
 
+    @authenticate
+    @require_perm("schedules", "update")
+    @dbsession
+    def patch(
+        self, schedule_name: str, token: AccessToken.Payload, session: so.Session
+    ):
+        """Update all properties of a schedule but _id and most_recent_task"""
+
+        schedule = dbm.Schedule.get(session, schedule_name, ScheduleNotFound)
+
+        try:
+            update = self._get_and_validate_patch(schedule=schedule, request=request)
```

Comment:
Not a fan of this kind of things. I guess Codefactor complained about complexity?

Reply:
Yes ... I always feel cheating when doing this ... 
But once I've done the modification I had to admit that it makes the code probably easier to read and definitely easier to test (we could test individual functions in unit tests instead of doing this in integration tests)